src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -173,6 +176,24 @@ export default class ResultsViewMutationTable extends MutationTable<
             };
         }
 
+        // generate namespace columns
+        const namespaceColumns = createNamespaceColumns(
```

Comment:
again it seems odd that we accept a prop of namespace columns only the mutate the columns prop.  seems like we should be doing this where the columns prop is prepared?